liquidator/index.js

```diff
@@ -1,17 +1,20 @@
 require("dotenv").config();
 
 // Helpers
-const { delay } = require("../financial-templates-lib/helpers/delay");
-const { Logger, waitForLogger } = require("../financial-templates-lib/logger/Logger");
 const { MAX_UINT_VAL } = require("../common/Constants");
 const { toBN } = web3.utils;
 
 // JS libs
 const { Liquidator } = require("./liquidator");
-const { GasEstimator } = require("../financial-templates-lib/helpers/GasEstimator");
-const { ExpiringMultiPartyClient } = require("../financial-templates-lib/clients/ExpiringMultiPartyClient");
-const { createReferencePriceFeedForEmp } = require("../financial-templates-lib/price-feed/CreatePriceFeed");
-const { Networker } = require("../financial-templates-lib/price-feed/Networker");
+const {
+  GasEstimator,
+  ExpiringMultiPartyClient,
+  Networker,
+  Logger,
+  createReferencePriceFeedForEmp,
+  waitForLogger,
+  delay
+} = require("@umaprotocol/financial-templates-lib");
 
 // Truffle contracts
 const ExpiringMultiParty = artifacts.require("ExpiringMultiParty");
```

Comment:
Is there a reason why these three contract imports are still using `artifacts.require()`, even though it seems we can import them via `require("@umaprotocol/core/build/contracts/x)`?

Reply:
I talked about this in the description. This refactor will be pretty painful (IMO), and I'd prefer to just let truffle do the work, especially in the tests.

Reply:
Ah I see now